lib/modules/apostrophe-templates/views/outerLayoutBase.html

```diff
@@ -16,6 +16,7 @@
 
     {% if data.user %}
       <div class="apos-ui">
+        {{ apos.modules['apostrophe-ui'].tooltip() }}
```

Comment:
Although this is elegant, since we're doing it 100% of the time and it takes no arguments, I recommend you just inline the div here as it avoids creating a whole 'nother nunjucks environment just to render the tiny template.

Reply:
Using an include statement is a middle ground that would still be faster than routing it through partial().